bin/flutter.bat

```diff
@@ -106,6 +107,9 @@ GOTO :after_subroutine
         SET PUB_ENVIRONMENT=%PUB_ENVIRONMENT%:flutter_bot
       :not_on_bot
       SET PUB_ENVIRONMENT=%PUB_ENVIRONMENT%:flutter_install
+      IF "%PUB_CACHE%" NEQ "" GOTO pub_cache_set
```

Comment:
Since this can be re-formulated to be an AND condition, I think nesting is easier to read here:
```
IF "%PUB_CACHE%" EQ "" (
 IF EXIST "%pub_cache_path%" SET PUB_CACHE=%pub_cache_path%
)
```